data/processors/maps.py

```diff
@@ -337,3 +337,37 @@ def _load_custom_maps():
             }
 
     return maps_out
+
+
+def add_overlay_maps(data):
+    """ Add the overlay maps to all entries where they apply """
+    with open("sources/46_overlay-maps.yaml") as f:
+        overlay_maps = yaml.safe_load(f.read())
+        
+    parent_lookup = {m["props"]["parent"]: m for m in overlay_maps}
+    parent_ids = set(parent_lookup.keys())
+    
+    for _id, entry in data.items():
+        candidates = set(entry["parents"]) & parent_ids
```

Comment:
Yeah, `&` might be confusing, because it isn't a union actually, but an intersection:
```python
{'a', 'b', 'c'} & {'b', 'c', 'd'}              # -> {'b', 'c'}
{'a', 'b', 'c'}.union({'b', 'c', 'd'})         # -> {'c', 'd', 'b', 'a'}
{'a', 'b', 'c'}.intersection({'b', 'c', 'd'})  # -> {'b', 'c'}
```

Reply:
<img src="https://media0.giphy.com/media/l3q2K5jinAlChoCLS/giphy.gif"/>